CHANGELOG.md

```diff
@@ -2,18 +2,91 @@
+  but we had to disable them. Now, the tests are going to be enforced on every component independently.
+  Hence, the camelCase config fields recently added with the new AWS Cloud Map resolver has to be fixed.
 
 ### 💡 Enhancements 💡
 
 - (Splunk) Include [`splunk-otel-dotnet`](https://github.com/signalfx/splunk-otel-dotnet) in the `splunk-otel-auto-instrumentation` deb/rpm packages (x86_64/amd64 only) ([#4679](https://github.com/signalfx/splunk-otel-collector/pull/4679))
   - **Note**: Only manual activation/configuration for .NET auto instrumentation is currently supported. See [README.md](https://github.com/signalfx/splunk-otel-collector/blob/main/instrumentation/README.md) for details.
+- (Splunk) Update splunk-otel-javaagent to `v1.32.0` ([#4715](https://github.com/signalfx/splunk-otel-collector/pull/4715))
+- (Splunk) Enable collecting MSI information on Windows in the support bundle ([#4710](https://github.com/signalfx/splunk-otel-collector/pull/4710))
+- (Core) `confighttp`: Disable concurrency in zstd compression ([#8216](https://github.com/open-telemetry/opentelemetry-collector/pull/8216))
+- (Core) `cmd/builder`: Allow configuring `confmap.Provider`s in the builder. ([#4759](https://github.com/open-telemetry/opentelemetry-collector/pull/4759))
+  If no providers are specified, the defaults are used.
+  The default providers are: env, file, http, https, and yaml.
+
+  To configure providers, use the `providers` key in your OCB build
+  manifest with a list of Go modules for your providers.
+  The modules will work the same as other Collector components.
+
+- (Core) `mdatagen`: enable goleak tests by default via mdatagen ([#9959](https://github.com/open-telemetry/opentelemetry-collector/pull/9959))
+- (Core) `cmd/mdatagen`: support excluding some metrics based on string and regexes in resource_attributes ([#9661](https://github.com/open-telemetry/opentelemetry-collector/pull/9661))
+- (Core) `cmd/mdatagen`: Generate config and factory tests covering their requirements. ([#9940](https://github.com/open-telemetry/opentelemetry-collector/pull/9940))
+  The tests are moved from cmd/builder.
+- (Core) `confmap`: Add `ProviderSettings`, `ConverterSettings`, `ProviderFactories`, and `ConverterFactories` fields to `confmap.ResolverSettings` ([#9516](https://github.com/open-telemetry/opentelemetry-collector/pull/9516))
+  This allows configuring providers and converters, which are instantiated by `NewResolver` using the given factories.
+- (Contrib) `vcenterreceiver`: Changes process for collecting VMs & VM perf metrics used by the `vccenterreceiver` to be more efficient (one call now for all VMs) ([#31837](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31837))
+- (Contrib) `splunkhecreceiver`: adding support for ack in the splunkhecreceiver ([#26376](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26376))
```

Comment:
Not sure if we should include this in our changelog since we don't have the ack extension in our distribution. We should probably be adding the ack extension.

Reply:
Adding the ack extension given that splunkhec can now work with it makes sense to me 👍

Reply:
PR posted: https://github.com/signalfx/splunk-otel-collector/pull/4724